jest.config.js

```diff
@@ -1,5 +1,8 @@
 const config = {
-  cacheDirectory: '<rootDir>/.cache/jest/'
+  cacheDirectory: '<rootDir>/.cache/jest/',
+  transform: {
+    '^.+\\.m?js$': ['babel-jest', { rootMode: 'upward' }]
```

Comment:
[Jest defaults to `babel-jest` anyway](https://jestjs.io/docs/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object) but we want it to find our Babel config